### ISSUE TITLE
tree-sitter-json allows only one object.

### ISSUE THREAD

**Issue by johnybx** [rest-nvim/tree-sitter-http#4]
In http grammar there is json grammar reused [here](https://github.com/NTBBloodbath/tree-sitter-http/blob/main/grammar.js#L3) but the problem is that this grammar allow only one object json object if I understand it correctly so then you have file with content like this:
```
GET http://{{API_URL}}/12345/path
Content-Type: application/json
Authorization: Bearer {{TOKEN}}

{
    "test": "blah"
}

# test
GET http://{{API_URL}}/12345/path
Content-Type: application/json
Authorization: Bearer {{TOKEN}}

{
    "test": "blah"
}
```
you get error from tree sitter:
```
request [0, 0] - [0, 27]
  method: method [0, 0] - [0, 3]
  url: url [0, 4] - [0, 27]
header [1, 0] - [1, 30]
  name: name [1, 0] - [1, 12]
  value: value [1, 14] - [1, 30]
header [2, 0] - [2, 31]
  name: name [2, 0] - [2, 13]
  value: value [2, 15] - [2, 31]
ERROR [4, 0] - [6, 1]
  object [4, 0] - [6, 1]
    pair [5, 4] - [5, 18]
      key: string [5, 4] - [5, 10]
        string_content [5, 5] - [5, 9]
      value: string [5, 12] - [5, 18]
        string_content [5, 13] - [5, 17]
comment [8, 0] - [8, 6]
request [9, 0] - [9, 34]
  method: method [9, 0] - [9, 3]
  url: url [9, 4] - [9, 34]
header [10, 0] - [10, 30]
  name: name [10, 0] - [10, 12]
  value: value [10, 14] - [10, 30]
header [11, 0] - [11, 31]
  name: name [11, 0] - [11, 13]
  value: value [11, 15] - [11, 31]
object [13, 0] - [15, 1]
  pair [14, 4] - [14, 18]
    key: string [14, 4] - [14, 10]
      string_content [14, 5] - [14, 9]
    value: string [14, 12] - [14, 18]
      string_content [14, 13] - [14, 17]
```

Any advice how this could be fixed ? I don't think json grammar should be updated but also just copying whole grammar to http does not see correct.

**Comment by johnybx** [comment]
I just found out that fix is pretty easy just changing:
```
    document: ($, original) => optional(original),
```
to
```
    document: ($, original) => repeat(original),
```
added that to https://github.com/NTBBloodbath/tree-sitter-http/pull/3

**Comment by NTBBloodbath** [comment]
Closed as #3 was merged. Thank you so much for digging on this! :)
